lib/whatsapp_sdk/api/messages.rb

```diff
@@ -411,6 +411,31 @@ def send_reaction(sender_id:, recipient_number:, message_id:, emoji:)
         Api::Responses::MessageDataResponse.build_from_response(response: response)
       end
 
+      # Send typing indicator
+      #
+      # @param sender_id [Integer] Sender' phone number.
+      # @param message_id [String] the id of the message received in the messages webhooks.
+      # @return [MessageDataResponse] Response object.
+      def send_typing_indicator(sender_id:, message_id:)
+        params = {
+          messaging_product: "whatsapp",
+          recipient_type: "individual",
+          status: "read",
+          message_id: message_id,
+          typing_indicator: {
+            type: "text"
+          }
+        }
+
+        response = send_request(
+          endpoint: endpoint(sender_id),
+          params: params,
+          headers: DEFAULT_HEADERS
+        )
+
+        Api::Responses::MessageDataResponse.build_from_response(response: response)
```

Comment:
```suggestion
        Api::Responses::SuccessResponse.success_response?(response: response)
```

The API returns `{ success: true }` when it succeeds.

Reply:
You're right. I've done the adjusts about it!